ModelCatalogueCorePlugin/src/groovy/org/modelcatalogue/core/util/Metadata.groovy

```diff
@@ -14,5 +14,6 @@ class Metadata {
      * The HPO and clinical tests report exporters only work for and should only be available to the relevant data class(es), discerned by this metadata field
      */
     public static final def HPO_REPORT_AVAILABLE = "rare-disease-hpo-clinical-tests-report"
+    public static final def CANCER_TYPES_AVAILABLE = "cancer-types-export"
```

Comment:
this applies to the HPO_REPORT_AVAILABLE, it's better to create the metadata keys as URLs where can be some explanation put in future